media-sound/amarok/amarok-9999.ebuild

```diff
@@ -13,13 +13,17 @@ HOMEPAGE="https://amarok.kde.org/"
 
 LICENSE="GPL-2"
 SLOT="5"
-IUSE="ipod lastfm mariadb mtp ofa podcast wikipedia"
+IUSE="cdrom ipod lastfm mariadb mtp ofa podcast wikipedia"
```

Comment:
IUSE=cdrom does not tick any buildsystem box. Optional RDEPEND IUSE are banned. What you can do instead is advertise it to the user via `optfeature.eclass` in a `pkg_postinst` message.